README.md

```diff
@@ -909,6 +927,32 @@ path, err = versions.CreatePath("subject/repo/pkg/version")
 btManager.MavenCentralContentSync(params, path)
 ```
 
+#### Using ContentReader
+
+Some APIs return a content.ContentReader which allow reading the API output.
+Here's an example for how it can be used:
+
+```go
+reader, err := servicesManager.SearchFiles(searchParams)
+if err != nil {
+    return err
+}
+defer reader.Close()
+for currentResult := new(ResultItem); reader.NextRecord(currentResult) == nil; currentResult = new(ResultItem)  {
+    fmt.Printf("Found artifact: %s of type: %s\n", searchResult.Name, searchResult.Type)
+}
+if err := resultReader.GetError(); err != nil {
+    return err
+}
+resultReader.Reset()
+```
+
+
+`reader.NextRecord(currentResult)` load the next record from the reader into `currentResult`  of type `ResultItem`.
+`reader.Close()` remove the results reader after it is used (preferably using `defer`).
+`reader.GetError()` any error that may accur during `NextRecord()`, can be return using `GetError()`.
+`reader.Reset()` after a successful reading of the whole reader, trying to read it again will not work and will not output any error, therefore `Reset()` is required before reading again. For best practice, always reset the reader after finish reading.
```

Comment:
I suggest the description to:

`reader.Reset()` sets the reader back to the begging of the input.